packages/v4/src/content/design-guidelines/components/tree-view/tree-view.md

```diff
@@ -3,5 +3,92 @@ id: Tree view
 section: components
 ---
 
-We’re working on some guidelines for this component.
-If you have any questions in the meantime, feel free to [get in touch with us.](/get-in-touch)
+A **tree view** is a structure that displays data in a hierarchical view. It can be used in a [dropdown](/components/dropdown), [drawer](/components/drawer), [primary-detail](/demos/primary-detail), [modal](/components/modal), or [wizard](/components/wizard).
+
+A default tree view provides a tree structure with items and an arrow to expand or collapse child items. Every tree view can be extended with these [optional elements](#tree-view-capabilities).
+
+## Tree view elements
+
+A default tree view consists of 5 elements:
+
+<img src="./img/tree-view-elements.png" width="332" />
+
+1. **Expand/collapse button:** A button that allows you to expand or collapse child nodes 
+2. **Parent node:** A node that contains other items (for example, a folder that contains multiple documents)
+3. **Child node:** A node within the parent node (for example, a document within a folder)
+4. **Leaf node:** The last item in a tree, which doesn't have any child nodes
+5. **Truncation indicator:** A control that automatically shortens long names 
+
+## Usage
+
+A tree view can be used for: 
+1. Changing the content on a page based on the selected item. 
+2. Selecting items or applying filters. 
+
+<img src="./img/Default-tree-view.png" width="1004" />
+
+Use a tree view when:   
+* Selecting is the main use-case. 
+* You apply filters that need to be structured in a clear hierarchy. 
+* The data is structured into levels.
+* You need to display hierarchies that have more than 2 levels.
+
+Don’t use a tree view when: 
+* The items are not structured hierarchically. Use the [data list](/components/data-list) instead. 
+* You select 1  item from a small set. Instead, use [select lists](/), [radio buttons](/components/radio), [checkboxes](/components/checkbox), or a [dropdown](/components/dropdown).
+* You need to see an overview of a large amount of data.
+* You need to compare two large sets of data. Use [charts](/charts/about) for a better view. 
+* The hierarchy is too deep and includes hundreds of items.
```

Comment:
What would someone use instead then? I feel like I would use a tree view for that but idk hahah

Reply:
Haha, I meant when the dataset is really small (5-10 items) and doesn't have a hierarchy, then you don't need a tree view. You can use just a checkbox or radio button etc. 

I think it's almost the same as the first point.

Reply:
I agree with that! That comment was actually about the last bullet though, "The hierarchy is too deep and includes hundreds of items"

Reply:
Oh :D Sometimes you have hundreds or thousands of items and the hierarchy can be really deep. I know that Allie had a use case that included so many items and then we agreed that would be better to use a drill-down menu instead of a tree view. I think it was merged yesterday. So, maybe we can mention it. WDYT?

Reply:
Hey sorry I didn't see your reply here until now^! Yes, I think mentioning it would be helpful, so like "the hierarchy is too deep and includes hundreds of items. Instead, use a drill-down menu." Thank you!! Also left a few other tiny comments, after this it should be good to go :)